internal_test/mySQL_test/database_test.go

```diff
@@ -1,42 +1,169 @@
 package mySQL_test
 
 import (
+	"database/sql"
+	_ "fmt"
+	"log"
+	"os"
 	"testing"
-
+	_ "github.com/go-sql-driver/mysql"
 	"github.com/matcha-devs/matcha/internal/mySQL"
 )
 
-var db = mySQL.Open()
+// This a variable to hold the test database connection,
+// and run all functions relating to the database:
+var test_db_connection *sql.DB
 
-// TODO(@SeoYoung Cho):what do is the panic error you are having?:
+// Mock environment variable for testing
+// Helper function to create a test database connection and ensure it's clean
+func setupTestDB(t *testing.T) *mySQL.Database {
+	// Get the password from the environment, currently using the admin password this may change in the future!!!
+	password := os.Getenv("MYSQL_PASSWORD")
+	// Connect to MySQL without specifying a database
+	var err error
+	test_db_connection, err = sql.Open("mysql", "root:"+password+"@tcp(localhost:3306)/")
```

Comment:
We only need to Open and Ping one time for the whole package, and then we can Execute before each test. Lastly, only Close once for the whole package.

Reply:
As discussed in the previous comment, this is a update feature mentioned in the PR.